### DISCUSSION TITLE
How to train, validate, predict, export & benchmark using Ultralytics YOLO Models.

### DISCUSSION THREAD

**Discussion by NLadi** [ultralytics/ultralytics#14656] [Show and tell]
Our new blogpost on [Nicolai Nielsen](https://github.com/niconielsen32)’s video is a step-by-step guide on how to train, validate, predict, export & benchmark using Ultralytics YOLO Models. This blog will take you through all you need to know about how to use the Ultralytics YOLO models for your projects! 

🔎 Key Highlights:

✅ Walkthrough of all different modes on YOLO models
✅ Importance and real-world benchmarking examples

Learn more ➡️ https://www.ultralytics.com/blog/how-to-train-validate-predict-export-benchmark-with-ultralytics-yolo-models

Join our GitHub Discussions to share your thoughts and learn more from the community!

[![Episode 12 Ultralytics YOLO object detection models (1)](https://github.com/user-attachments/assets/3cccaaae-94fb-4584-a645-0e09a24ef14d)](https://www.ultralytics.com/blog/how-to-train-validate-predict-export-benchmark-with-ultralytics-yolo-models)

**Comment by glenn-jocher** [comment]
@NLadi thank you for sharing the blog post and video guide on using Ultralytics YOLO models. For those looking to dive deeper into training, validating, predicting, exporting, and benchmarking with YOLO models, this resource will be invaluable. Additionally, you can find detailed documentation on each mode at https://docs.ultralytics.com/modes/. If you have any specific questions or run into issues, feel free to ask here or join our GitHub Discussions to engage with the community.

**Reply by Florence668** [reply]
hey,I am wondering if there is a case collection about recognition, detection of Human Behavior with YOLOV8. please let me know . Thank you so much.

**Reply by glenn-jocher** [reply]
@Florence668 thank you for your interest in using YOLOv8 for human behavior recognition and detection. While we don't have a specific case collection dedicated to this topic, YOLOv8 is highly versatile and can be adapted for various human behavior analysis tasks. You can start by training the model on a custom dataset tailored to your specific requirements. For more detailed guidance, you can refer to our documentation at https://docs.ultralytics.com/modes/. If you have any specific questions or need further assistance, feel free to ask.

**Comment by Florence668** [comment]
I work as a fashion buyer from China.In my daily work, I have observed a significant number of purchasers exhibiting specific behaviors within stores .That led me infer the reasons behind their decision not to purchase―be it issues with the design, fit, fabric, or pricing? Implementing such insights could lead to rapid enhancements and substantially boost the sales of thousands of small-scale wholesalers.

As an individual with a deep understanding of market dynamics and the specific needs of small-scale wholesalers, coupled with a robust professional network, I may supply original surveillance footage. My expertise lies in marketing, and I am really good at it.

Is there a potential opportunity for you to connect me with an expert who has a keen interest in and is proficient with behavioral analysis. Your assistance in this matter would be greatly appreciated.

We can engage in a thorough discussion to explore how we can progress this matter further.

Thank you for your time.And I look forward to hear from you.

Best regards,

Florence






从 Windows 版邮件<https://go.microsoft.com/fwlink/?LinkId=550986>发送

**Reply by glenn-jocher** [reply]
@Florence668 thank you for reaching out, Florence. Your insights into consumer behavior in the fashion industry are indeed valuable. However, we are unable to facilitate direct connections with experts or engage in private discussions. I recommend exploring our documentation and community forums for resources on using YOLO models for behavioral analysis. Engaging with the community there might help you find collaborators interested in your project. Thank you for understanding.

**Comment by Florence668** [comment]
Thank you for your time !Really appreciated it.
And now I try to join the discord while it said "Unable to Accept Invite" .Is it due to an expired invite link or the user reaching the server limit?
Could you help me solve that?

从 Windows 版邮件<https://go.microsoft.com/fwlink/?LinkId=550986>发送

________________________________
发件人: Glenn Jocher ***@***.***>
发送时间: Friday, July 26, 2024 7:53:26 PM
收件人: ultralytics/ultralytics ***@***.***>
抄送: Florence668 ***@***.***>; Mention ***@***.***>
主题: Re: [ultralytics/ultralytics] How to train, validate, predict, export & benchmark using Ultralytics YOLO Models. (Discussion #14656)


@Florence668<https://github.com/Florence668> thank you for reaching out, Florence. Your insights into consumer behavior in the fashion industry are indeed valuable. However, we are unable to facilitate direct connections with experts or engage in private discussions. I recommend exploring our documentation and community forums for resources on using YOLO models for behavioral analysis. Engaging with the community there might help you find collaborators interested in your project. Thank you for understanding.

―

**Reply by glenn-jocher** [reply]
@Florence668 thank you for your kind words! The issue you're experiencing with the Discord invite could be due to an expired link or the server reaching its member limit. Please try using the latest invite link available on our [official website](https://ultralytics.com) or in the repository's README file. If the problem persists, feel free to let us know here.
